src/containers/now-playing/NowPlaying.tsx

```diff
@@ -292,6 +294,15 @@ const NowPlaying = g(
     ])
 
     const onPrevious = () => {
+      if (track.genre === Genre.PODCASTS) {
+        const position = timing.position
+        const newPosition = position - SKIP_DURATION_SEC
+        if (newPosition > 0) {
```

Comment:
should we use seek(Math.mix(0, newPosition)) and not do this check

Reply:
subtle difference -- if newPosition is <= 0, instead, we want to not return and fall through, so i think this is best. thoughts?

Reply:
nvm u right